scst-store/tls-configuration.hbs.md

```diff
@@ -0,0 +1,69 @@
+
+# Setting up custom TLS settings in the Store
+
+###  <a id="setting_up_tls_namespace_and_secretName"></a>Setting up custom ingres tls certificate
+
+In the `tap-values.yaml` file, `namespace` and `secretName`  can be set as shown below:
+
+---
+```yaml
+tls:
+  namespace: "namespace"
+  secretName: "secretName"
+```
+---
+---
+```
+`namespace` : The targeted namespace for secret consumption by the HTTPProxy. 
+`secretName` : The name of secret for consumption by the HTTPProxy.
+```
+---
+  ##### Note: The certificate secret should be created before deploying the Store. By default Store will be setup with self signed certificate when ingres is enabled.
```

Comment:
ingres should be ingress

Reply:
"By default the Store will be setup with a self signed certificate when ingress is enabled."